test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/CompanyDetailsControllerSpec.scala

```diff
@@ -2227,6 +2227,110 @@ class CompanyDetailsControllerSpec
 
     }
 
+    "handling submitted answers to the multiple multiple Indirect Disposals Guidance" must {
+
+      def performAction(formData: (String, String)*): Future[Result] =
+        controller.multipleIndirectDisposalsGuidance()(
+          FakeRequest().withFormUrlEncodedBody(formData: _*).withMethod("POST")
+        )
+
+      behave like redirectToStartBehaviour(() => performAction())
+
+      behave like amendReturnToFillingOutReturnSpecBehaviour(
+        controller.multipleIndirectDisposalsGuidance(),
+        mockUUIDGenerator
+      )
+
+      "not update the session" when {
+
+        "the data submitted is the same as one that already exists in session" in {
+
+          val disposalPrice = AmountInPence.fromPounds(1000)
+
+          inSequence {
+            mockAuthWithNoRetrievals()
+            mockGetSession(
+              SessionData.empty.copy(
+                journeyStatus = Some(
+                  sample[FillingOutReturn].copy(
+                    draftReturn = sample[DraftMultipleIndirectDisposalsReturn].copy(
+                      exampleCompanyDetailsAnswers = Some(
+                        sample[IncompleteExampleCompanyDetailsAnswers].copy(
+                          disposalPrice = Some(disposalPrice)
+                        )
+                      )
+                    ),
+                    subscribedDetails = sample[SubscribedDetails].copy(
+                      name = Right(sample[IndividualName])
+                    )
+                  )
+                )
+              )
+            )
+          }
+
+          checkPageIsDisplayed(
+            controller.multipleIndirectDisposalsGuidance()(FakeRequest()),
+            messageFromMessageKey("company-details.multiple-indirect-disposals.guidance.title")
+          )
+        }
+      }
+
+    }
+
+    "handling submitted answers to the multiple indirect disposal guidance page" must {
```

Comment:
This looks same as above, whats the difference?

Reply:
These are testing two different functions multipleIndirectDisposalsGuidance and multipleIndirectDisposalsGuidanceSubmit